docs/source/workflow_inputs.rst

```diff
@@ -310,6 +310,102 @@ Default values are from *Table 5.1.2(1) National Average Emission Factors for Ho
+  ``MarginalRate``                  double    $/therm               No        See [#]_  Utility rate marginal rate
+  ================================  ========  =======  ===========  ========  ========  ============================================================
+
+  .. [#] If MarginalRate not provided, defaults to state, regional, or national average that can be found at ``ReportUtilityBills/resources/Data/UtilityRates/NG_PRI_SUM_A_EPG0_PRS_DMCF_A.csv``.
+
+Propane and Fuel Oil Utility Rates
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For each scenario, propane and fuel oil utility rates can be optionally entered as an ``/HPXML/SoftwareInfo/extension/UtilityBillScenarios/UtilityBillScenario/UtilityRate``.
+
+  ================================  ========  ========  ===========  ========  ========  ============================================================
+  Element                           Type      Units     Constraints  Required  Default   Notes
+  ================================  ========  ========  ===========  ========  ========  ============================================================
+  ``FuelType``                      string              See [#]_     Yes                 Utility rate fuel type
+  ``MarginalRate``                  double    $/gallon               No        See [#]_  Utility rate marginal rate
+  ================================  ========  ========  ===========  ========  ========  ============================================================
+
+  .. [#] FuelType choices are "propane" and "fuel oil".
+  .. [#] If MarginalRate not provided, defaults to state, regional, or national average that can be found at ``ReportUtilityBills/resources/Data/UtilityRates/PET_PRI_WFR_A_EPLLPA_PRS_DPGAL_W.csv`` (propane) or ``ReportUtilityBills/resources/Data/UtilityRates/PET_PRI_WFR_A_EPD2F_PRS_DPGAL_W.csv`` (fuel oil).
+
+Other Fuel Utility Rates
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+For each scenario, other fuel utility rates can be optionally entered as an ``/HPXML/SoftwareInfo/extension/UtilityBillScenarios/UtilityBillScenario/UtilityRate``.
+
+  ================================  ========  ======  ===========  ========  ========  ============================================================
+  Element                           Type      Units   Constraints  Required  Default   Notes
+  ================================  ========  ======  ===========  ========  ========  ============================================================
+  ``FuelType``                      string            See [#]_     Yes                 Utility rate fuel type
+  ``MarginalRate``                  double    $/kBtu               Yes                 Utility rate marginal rate
+  ================================  ========  ======  ===========  ========  ========  ============================================================
+
+  .. [#] FuelType choices are "coal", "wood", and "wood pellets".
```

Comment:
This seems more complicated than it needs to be. I'll take a crack at simplifying it.

Reply:
Just FYI part of the motivation for splitting things up is to support the eventual changes with "detailed" utility bill calculations.

Reply:
Ah, good point. So electricity needs to be broken out separately.